README.md

```diff
@@ -1 +1,172 @@
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+# 📖 Book-store-App <a name="Book-store"></a>
+
+## HTML+CSS Setup
+
+**Math-magician** is simple single page web application that servers as book store application built with React. It allows users to view, add, and delete books in a collection.
+
+## 🛠 Built With <a name="built-with"></a>
+This project is built with:
+
+- ![React](https://img.shields.io/badge/-React-000000?style=flat&logo=react)
+- ![JavaScript](https://img.shields.io/badge/-JavaScript-000000?style=flat&logo=javascript)
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>JSX</li>
+    <li>React </li>
+   
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Key Features
+- **Single Page Application**
+- **Book storage**
+- **Navigation and deletion of a book**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get the content of this application project locally you need to runfollow  these step in your terminal:
+
+- `cd book-store`
+- `npm install` To download all the dependancies of the project
+- Use `npm start` to run the app in your browser
+
+### Prerequisites
+
+In order to run this project you need:
+
+<ul>
+  <li>Vs code</li>
+  <li>npm packages</li>
+  <li>Install React</li>
+</ul>
+
+
+### Setup
+
+Clone this repository to your desired folder: 
+```sh
+
+cd my-dir
+git clone https://github.com/Christelle-12/book-store.git
+
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Install
+
+1. Clone the repo:https://github.com/Christelle-12/book-store.git
+ 
+2. Install npm packages:
+  npm install
+3. Install webpack:
+  webpack.config.js
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Usage
+
+To run tests, run the following command:
+
+<ul>
+<li>run npm start</li>
+<!-- <li>run npm build</li>
+<li>npm run dev </li> -->
+</ul>
+
+### Run Tests
+
+to run tests, run the following command
+
+<ul>
+  <li>npx stylelint "**/*.{css,scss}"</li>
+  <li> npx eslint "**/*.{js,jsx}"</li>
+  <!-- <li>npx hint .</li> -->
+</ul>
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author**
+
+
+👤 Nirere Marie Christelle
+- GitHub: [@Christelle-12](https://github.com/Christelle-12)
+- Twitter: [@Chr1Nirere](https://twitter.com/Chr1Nirere)
+- LinkedIn :[LinkedIn](https://www.linkedin.com/in/nirere-marie-christelle-9b139823b/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## Future Features
+
+- [ ] **[Add Transitions using CSS]**
+- [ ] **[Add stylesheet]**
+- [ ] **[Add Css animations]**
+- [ ] **[Add Animations Features]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page].
```

Comment:
- [ ] To make the `[issues page]` a link we can add `()` right after the closing `]` then we insert our link

Reply:
- [ ] [OPTIONAL] Great effort trying to implement this, let me add additional explanations; 
In order to make the `[issues page]` a link, you need to do this:
`[Issues Page](https://github.com/Christelle-12/book-store/issues)` which will present it as a link like this [Issues Page](https://github.com/Christelle-12/book-store/issues) when you preview it.